internal/services/server.go

```diff
@@ -30,6 +28,10 @@ const (
 	// V1SchemaServiceEnabled indicates that the V1 schema service is enabled.
 	V1SchemaServiceEnabled SchemaServiceOption = 1
 
+	// V1SchemaServiceAdditiveOnly indicates that the V1 schema service is enabled in additive-only
+	// mode for testing.
+	V1SchemaServiceAdditiveOnly SchemaServiceOption = 2
```

Comment:
Can we switch this section to use `iota`?

Reply:
I thought general consensus lately was not to use it, lest you accidentally add one in between and change it?

Reply:
Ah, that's a pretty good argument. I've never seen it happen, though.